assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.xml

```diff
@@ -592,12 +1125,121 @@
+            <summary>
+            Constructs a <see cref="T:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorSubscribeResponseProcessor" />.
+            </summary>
+            <param name="logger">The logger that the consumer has configured.</param>
+            <param name="dataMapper">The DataMapper that will take care of storing the incoming data.</param>
+            <param name="origin">Where the data originates from.</param>
+            <param name="name">
+            The name of the processor. This will be either the name of the subscription or <see cref="F:System.String.Empty"/> for the processor that will take care
+            of the data that came in through a <see cref="T:Gnmi.GetResponse" />. 
+            </param>
+            <param name="action">The handler that the consumer has configured to process the messages.</param>
+            <param name="subscriptionMode">The mode of the subscription.</param>
+            <exception cref="T:System.ArgumentNullException">Throws when one of the required arguments is null.</exception>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorSubscribeResponseProcessor.Process(Gnmi.SubscribeResponse)">
+            <summary>
+            Processes a <see cref="T:Gnmi.SubscribeResponse" /> and will apply the changes to the in-memory data model.
+            </summary>
+            <param name="subscribeResponse">The subscribe response that needs to be processed.</param>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorSubscribeResponseProcessor.FlushChanges(System.Object)">
+            <summary>
+            Goes over all the changes that have been applied to the in-memory model and flushes the data to SLProtocol.
+            </summary>
+            <param name="state">Unused, needs to match the delegate.</param>
+        </member>
+        <member name="T:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorUtils">
+            <summary>
+            Class that contains common shared methods of the DataMinerConnector classes.
+            </summary>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorUtils.PassRawValue(Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.Args.DataMinerConnectorRawValueArgs)">
+            <summary>
+            Default method to be executed when no external method has been provided. This method simply returns the value that was present in the argument, except for byte[] objects which are serialized as string.
```

Comment:
Do you mean "except for those byte[] objects that are serialized as string", or "except for byte[] objects, and these are serialized as string"?